src/install.rst

```diff
@@ -6,148 +6,184 @@ Installing Nextstrain
 
-         Setting default environment to native.
+         .. code-block:: bash
 
-      Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-      Next, try :doc:`tutorials/quickstart`.
+            nextstrain check-setup --set-default
 
-   .. group-tab:: Docker
+         The final output from the last command should look like this, where ``<option>`` is the environment chosen in the previous step:
 
-    `Docker <https://docker.com/>`_ is a container system freely-available for all platforms.
-    When you use the Nextstrain CLI with Docker, you don’t need to manage any other Nextstrain software dependencies as validated versions are already bundled into `a container image by the Nextstrain team <https://github.com/nextstrain/docker-base/>`_.
+         .. code-block:: none
 
-    First, `follow Docker's installation guide <https://docs.docker.com/engine/install/>`_ for your operating system.
-    After installing and starting Docker, create a Conda environment named ``nextstrain``.
-    This command will install the Nextstrain CLI and Git (a dependency of subsequent tutorials).
+            Setting default environment to <option>.
 
-    .. code-block:: bash
+   .. group-tab:: Windows
 
-        conda create -n nextstrain -c conda-forge -c bioconda nextstrain-cli git
+      .. note::
 
-    Confirm that the installation worked and configure the CLI to use Docker as the default environment manager.
+         Due to installation constraints, there is no way to use the native build/view environment on Windows directly. Follow steps for **WSL on Windows** if the native environment is desired.
 
-    .. code-block:: bash
+      1. Install `Anaconda or Miniconda <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html>`_.
+         Miniconda is sufficient for this guide.
+      2. Install Visual C++ build tools following `this guide <https://stackoverflow.com/a/64262038>`_.
 
-        conda activate nextstrain
-        nextstrain check-setup --set-default
+         - This is necessary for a dependency of Nextstrain CLI and `we are investigating options to eliminate this step <https://github.com/nextstrain/cli/issues/31#issuecomment-970641263>`_.
 
-    The final output from the last command should look like this:
+      3. Open an Anaconda Prompt, which can be found in the Start menu.
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        Setting default environment to docker.
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-    Finally, download the latest Docker image for Nextstrain.
+      5. Install the Nextstrain build/view environment by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
```

Comment:
Both Docker and Microsoft documentation sites are written for highly technical users. I know all of the details our users will need to install Docker are in those docs, for example, but just parsing those docs could be painful for someone without much technical experience (i.e., our Windows users).

I don't know how much we want to go down this path, but one option to alleviate this pain would be for us to provide our own parallel installation guide with screenshots and less technical prose. I'm not recommending that approach for this PR, but we should see how complicated our users actually find this experience and keep an open mind about going this direction.

Reply:
My impression was that [Singularity](https://www.blopig.com/blog/2021/09/using-singularity-on-windows-with-wsl2/) was easier to install on Windows than Docker (singularity can pull and run Docker images, without requiring sudo. And HPC clusters usually have singularity but no docker). Just a discussion point, focusing on Docker is fine.

Reply:
I'll note that we don't currently support Singularity as a runtime for the CLI, though it's a [long standing todo](https://github.com/nextstrain/cli/issues/2). On Windows, Singularity will still need to run in a VM though, which means managing that directly unless Singularity packages tools to do that automatically (e.g. like Docker Desktop does).

Reply:
I added explicit step-by-step instructions for Miniconda in https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/94beb5c0c0973a012a693ad28119a72c2738c7be and started doing the same for Docker/WSL, but didn't include those since it's complex and I don't want to hinder the brevity of our docs.

Will keep this as-is, but a separate installation guides with screenshots as @huddlej mentioned is worth considering later based on user feedback.